### PR TITLE
MWPW-127701 CaaS Publish Date

### PR DESCRIPTION
**Pull request by chrischrischris**
Add support for `publishdate` metadata.
Adds a check for dates in day-month-year format

Resolves: [MWPW-127701](https://jira.corp.adobe.com/browse/MWPW-127701)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://c3-caaspublishdate--milo--adobecom.hlx.page/?martech=off

### CODE REVIEW

File: tools/send-to-caas/send-utils.js [hunk 1]
```diff
@@ -249,6 +249,14 @@ const getTags = (s) => {
 const getDateProp = (dateStr, errorMsg) => {
   if (!dateStr) return undefined;
   try {
+    if (dateStr.length === 10) {
+      const [m, d, y] = dateStr.split('-');
+      if (m > 12) {
```
Comment (rgclayton) [inline tools/send-to-caas/send-utils.js:254]: Month greater than 12?

Comment (meganthecoder) [reply tools/send-to-caas/send-utils.js:254]: That means that some dates will be misleading, but I suppose you don't have any way of knowing?

Comment (chrischrischris) [reply tools/send-to-caas/send-utils.js:254]: Yep.  No way of knowing.  The alternative is to fail and force authors to fix it in the metadata.

**Review by meganthecoder — APPROVED** [review]

File: tools/send-to-caas/send-utils.js (hunk 1 above)
Comment (rgclayton) [reply tools/send-to-caas/send-utils.js:254]: Got it. Thanks for the explanation.

**Review by rgclayton — APPROVED** [review]

**Review by sukamat — APPROVED** [review]

**Pull request merged**